src/diffbot.js

```diff
@@ -261,10 +265,12 @@ class Diffbot {
           throw new Error('missing name');
         else if (!options.seeds || !options.seeds.length)
           throw new Error('missing seeds');
-
+        let websites = options.seeds.split(' ');
+        const maxToCrawl = websites.length*1000;
+        const maxToProcess = maxToCrawl;
         let diffbot_url = `https://api.diffbot.com/v3/crawl?token=${this.token}`
           + `&name=${encodeURIComponent(options.name)}`
-          + `&seeds=${encodeURIComponent(options.seeds.join(' '))}`;
+          + `&seeds=${encodeURIComponent(options.seeds)}`;
```

Comment:
```suggestion
          + `&seeds=${encodeURIComponent(options.seeds.join(' '))}`;
```
Still need this if `seeds` is an array.